code/modules/clothing/gloves/color.dm

```diff
@@ -22,6 +22,20 @@
 	..()
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
+/obj/item/clothing/gloves/color/old
+	desc = "Old and worn out insulated gloves, hopefully they still work."
+	name = "worn out insulated gloves"
+	icon_state = "yellow"
+	item_state = "ygloves"
+	siemens_coefficient = 1
+	permeability_coefficient = 0.05
+	item_color="yellow"
+	resistance_flags = 0
+
+/obj/item/clothing/gloves/color/old/New()

```

Comment:
It's a function we call on objects immediately once map is fully loaded and ready. Replace New with Initialize, we have more control over initialization times and process with it. See https://github.com/tgstation/tgstation/blob/master/code/controllers/subsystem/atoms.dm if you want more details.